Misc/NEWS.d/next/Library/2020-11-10-07-04-15.bpo-40988.5kBC-O.rst

```diff
@@ -0,0 +1,2 @@
+Improve performance of :class:`functools.singledispatchmethod` by caching the generated dispatch wrapper.
+Optimization suggested by frederico. Patch by @mental32, @alexwaygood and @eendebakpt.
```

Comment:
- Wrap docs to 80 lines
- We usually use people's full names in changelogs, not their GitHub usernames

```suggestion
Improve performance of :class:`functools.singledispatchmethod` by caching the
generated dispatch wrapper. Optimization suggested by frederico. Patch by
@mental32, Alex Waygood and Pieter Eendebak.
```